common/spec/dependabot/dependabot_error_spec.rb

```diff
@@ -0,0 +1,217 @@
+# typed: false
+# frozen_string_literal: true
```

Comment:
Any reason why the file is renamed?

Reply:
I'd guess because of https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/FileName

Reply:
thanks Jurre